### ISSUE TITLE
Cacti Version 1.1.35 - Report Problem

### ISSUE THREAD

**Issue by anarkia1976** [Cacti/plugin_reportit#13]
I can create report without problem, but when i try to send by email i have this error:

2018/02/21  08:45:40 - CMDPHP PHP ERROR NOTICE Backtrace: (/reports_admin.php: 47  reports_send)(/lib/html_reports.php: 819  CactiErrorHandler)(/lib/functions.php: 4624 cacti_debug_backtrace)
2018/02/21 08:45:40 - ERROR PHP NOTICE: Undefined variable:  reports_item in file: /usr/share/cacti/lib/html_reports.php  on line:  819
2018/02/21 08:45:40 - MAILER Mail Successfully Sent to 'xxx@xxx.it', Subject: 'XXX - XXXX'

I never received email even the log message says the opposite.

thanks a lot.

**Comment by cigamit** [comment]
ReportIT is not officially supported yet.  If you are curious check out the pull request in the legacy branch.  Sorry about that.

**Comment by anarkia1976** [comment]
@cigamit no problem, thanks a lot for your great support.

**Comment by cigamit** [comment]
Please wait a few days. We are waiting on a user to re-submit a pull request. In the mean time watch the repo for the pull request and merge.

**Comment by cigamit** [comment]
We had a real mess to clean up.  Should be better in a few days.

**Comment by cigamit** [comment]
So, the legacy version that is 1.x compatible has been merged into develop.  Close if this specific problem is resolved.  Open new issues separately.

**Comment by anarkia1976** [comment]
closed.
